### ISSUE TITLE
ChatOllama stream method raises warn_deprecated NotImplementedError

### ISSUE THREAD

**Issue by v-byte-cpu** [langchain-ai/langchain#14980]
### System Info

langchain version: v0.0.352
python version: 3.11

Hi there! After that PR https://github.com/langchain-ai/langchain/pull/14713 was merged, I started getting errors in stream() method:
```
File .../lib/python3.11/site-packages/langchain_core/_api/deprecation.py:295, in warn_deprecated(since, message, name, alternative, pending, obj_type, addendum, removal)
    293 if not removal:
    294     removal = f"in {removal}" if removal else "within ?? minor releases"
--> 295     raise NotImplementedError(
    296         f"Need to determine which default deprecation schedule to use. "
    297         f"{removal}"
    298     )
    299 else:
    300     removal = f"in {removal}"

NotImplementedError: Need to determine which default deprecation schedule to use. within ?? minor releases
```

I guess this decorator must have a `pending=True` argument.

### Who can help?

@hwchase17 @agola11 

### Information

- [ ] The official example notebooks/scripts
- [X] My own modified scripts

### Related Components

- [X] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

```
from langchain.chat_models import ChatOllama
llm = ChatOllama(
    model="openchat:7b-v3.5-1210-q4_K_M",
)
for chunk in llm.stream("Where were the Olympics held?"):
    print(chunk, end="", flush=True)
```

### Expected behavior

successful streaming output from llm

**Comment by tylertitsworth** [comment]
I'm getting the same issue in my app. Chainlit offers a generic langchain provider that streams the model backend request provided by an llm input. When I add support for updating slider inputs in a playground request like so:

```python
result = await make_async(self.llm.stream)(
    input=messages, # User Prompt
    **request.prompt.settings # User Inputs like temperature, top_K, etc.
)
```

I get the error above. 

When I downgrade to v0.0.350 I also receive the same error, so I am unsure if the issue is in the specific PR you mentioned.

The out-of-the-box `GenericLangchainProvider` in chainlit for v0.0.352 doesn't even work.

https://github.com/langchain-ai/langchain/assets/43555799/827dfcfd-d66f-42ec-aa47-c4ae1df29bbf

**Comment by romellfudi** [comment]
The issue still persists in the latest version. Please provide guidance on how to handle streaming in ChatOllama.

**Comment by tylertitsworth** [comment]
Issue was fixed in #15104

**Comment by kesavan22** [comment]
I'm facing this issue for GooglePalm in the latest version
